dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -74,6 +74,7 @@ hadoop-annotations/2.7.4//hadoop-annotations-2.7.4.jar
 hadoop-auth/2.7.4//hadoop-auth-2.7.4.jar
 hadoop-aws/2.7.4//hadoop-aws-2.7.4.jar
 hadoop-azure/2.7.4//hadoop-azure-2.7.4.jar
+hadoop-client-api/3.3.1//hadoop-client-api-3.3.1.jar
```

Comment:
Surprisingly, when I update Jackson, this appears. I will investigate this separately.